command.go

```diff
@@ -993,7 +1046,20 @@ func (c *Command) HasAlias(s string) bool {
 	return false
 }
 
-// NameAndAliases returns string containing name and all aliases
+// HasNameOrAliasPrefix returns true if the Name or any of aliases start
+// with prefix
+func (c *Command) HasNameOrAliasPrefix(prefix string) bool {
```

Comment:
Why do we need it public? It's better to make it private for having in API only necessary methods.

Reply:
ok, makes sense. I don't remember why I exported this one

Reply:
ok, makes sense. I don't remember why I exported this one